data-explorer/kusto/query/mvexpandoperator.md

```diff
@@ -87,6 +93,20 @@ datatable (a:int, b:dynamic, c:dynamic)[1,dynamic({"prop1":"a", "prop2":"b"}), d
 |1|{"prop1":"a"}|5|
 |1|{"prop2":"b"}|5|
 
+## Convert output
+
+If you want to force the output of an mv-expand to a certain type (default is dynamic), use *to typeof*:
+
+<!-- csl: https://help.kusto.windows.net:443/Samples -->
+```kusto
+range x from 1 to 4 step 1
+| summarize x = make_list(x)
+| mv-expand with_itemindex=Index x
```

Comment:
The example is not matching the description (covert output)

Reply:
I changed that.